Sniffs/Commenting/PropertyCommentSniff.php

```diff
@@ -94,7 +94,9 @@ protected function processTokenWithinScope(
             array(T_DOC_COMMENT_OPEN_TAG),
             $stackPtr
         );
-        if ($postComment !== false && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']) {
+        if ($postComment !== false
+            && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']
```

Comment:
PHPCS code style wants the boolean operator on the new line:
Each line in a multi-line IF statement must begin with a boolean operator